### ISSUE TITLE
CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler

### ISSUE THREAD

**Issue by cesarazuero** [theappbusiness/TABTestKit#157]
Hi, I'm running the tests in Xcode 13 and I'm getting this error in the TABTestKit Framework.

`CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'TABTestKit' from project 'Pods')
    cd /Users/cesarazuero/Documents/ios-mobile-application/Pods
    export DEVELOPER_DIR\=/Applications/Xcode.app/Contents/Developer
    export SDKROOT\=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name TABTestKit -Onone -enable-batch-mode -enforce-exclusivity\=checked @/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/Objects-normal/x86_64/TABTestKit.SwiftFileList -DDEBUG -D COCOAPODS -suppress-warnings -sdk /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk -target x86_64-apple-ios13.0-simulator -g -module-cache-path /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -Xfrontend -serialize-debugging-options -profile-coverage-mapping -profile-generate -enable-testing -index-store-path /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Index/DataStore -swift-version 5 -I /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Products/Debug-iphonesimulator/TABTestKit -I /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/lib -F /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Products/Debug-iphonesimulator/TABTestKit -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/Library/Frameworks -F /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator15.0.sdk/Developer/Library/Frameworks -c -j8 -output-file-map /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/Objects-normal/x86_64/TABTestKit-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/Objects-normal/x86_64/TABTestKit.swiftmodule -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/swift-overrides.hmap -Xcc -iquote -Xcc /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/TABTestKit-generated-files.hmap -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/TABTestKit-own-target-headers.hmap -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/TABTestKit-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/all-product-headers.yaml -Xcc -iquote -Xcc /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/TABTestKit-project-headers.hmap -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Products/Debug-iphonesimulator/TABTestKit/include -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/DerivedSources-normal/x86_64 -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/DerivedSources/x86_64 -Xcc -I/Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG\=1 -Xcc -DDEBUG\=1 -Xcc -DCOCOAPODS\=1 -emit-objc-header -emit-objc-header-path /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/Objects-normal/x86_64/TABTestKit-Swift.h -import-underlying-module -Xcc -ivfsoverlay -Xcc /Users/cesarazuero/Library/Developer/Xcode/DerivedData/ShellEnergy-gthtgyantxmhcwfaxzikrfmjzuye/Build/Intermediates.noindex/Pods.build/Debug-iphonesimulator/TABTestKit.build/unextended-module-overlay.yaml -working-directory /Users/cesarazuero/Documents/ios-mobile-application/Pods
`

**Comment by theblixguy** [comment]
Does Xcode provide you with any more information about the problem? The above does not include any information about what's going on (it just lists the flags passed to the build).

**Comment by cesarazuero** [comment]
No, sorry, doesn't gives more info. Is there any restriction that I have to have in mind in the Podfile? Any minimum platform ?

**Comment by theblixguy** [comment]
If you dig into it you should be able to find some more information about the error. Have you checked the Xcode build log? (sometimes the info is hidden and you have to tap "more info" or something to "expand" the log and show the details).

**Comment by theblixguy** [comment]
> Is there any restriction that I have to have in mind in the Podfile? Any minimum platform ?

You need to be on iOS 12.4 or higher, and Swift 5 or higher.
